docs/adrs/0077-permanently-store-origin-duty-gbloc.md

```diff
@@ -0,0 +1,98 @@
+---
+title: 0077 Permanently store Origin Duty Location GBLOC
+description: |
+  Decision outcome: [TBD]
+---
+
+# Permanently Store the Origin Duty Location GBLOC
+
+**User Story:** *[MB-15032](https://dp3.atlassian.net/browse/MB-15032)* :lock:
+
+## Problem Statement
+
+In our current database structure, GBLOCs are stored in the `postal_code_to_gblocs` as a one-to-many value - one GBLOC can 
+represent many postal codes - and MTOs associate to GBLOCs using the `origin_duty_location_to_gbloc` view.
```

Comment:
Using MTO here could be ambiguous because the origin duty location is first chosen by the customer prior to the move being submitted and can be changed by a services counselor prior to the TOO making the move "available to the prime"/"creating the move task order" itself. In the office UI nothing displays on the Move Task Order page until a TOO has approved the move and at least one shipment.

Reply:
Ah yes, I see what you mean @duncan-truss . I'll revise some of the language here for clarity. Thank you!